client/src/components/PostPage/PostPageButtons/PostPageButtons.test.js

```diff
@@ -1,9 +1,9 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import { PostPageButtons } from "./PostPageButtons";
+// import React from "react";
```

Comment:
travis ci cant read the window (useParams accesses window API) so it crashes. will have to research how to mock that